### DISCUSSION TITLE
AWS Client VPN

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#61] [General]
# AWS Client VPN

Check if AWS Client VPN is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/c36b5ccf-28cf-4184-bd73-874171fb2442

**Comment by luismiparra** [comment]
update:
Because the only missing part to work is the driver I discover it just work by installing the oficial openvpn arm64 client on windows (in addition of the AWS vpn client) . This install the driver and make AWS vpn works. 

https://openvpn.net/community-downloads/

**Reply by OpenSource03** [reply]
Updated, thanks.

**Comment by Ascension-Tech** [comment]
The workaround ONLY works with AWS VPN Client version 4.x.x
My findings:

Windows 11 Pro (ARM64) on Snapdragon
✅ AWS VPN Client 4.x.x + OpenVPN ARM64 = Works perfectly
❌ AWS VPN Client 5.x.x + OpenVPN ARM64 = Does NOT work

What happens with version 5.x.x:

Even with OpenVPN ARM64 TAP driver installed, AWS VPN Client 5.x.x fails to recognize or use it
Fresh install of 5.x.x doesn't work
Upgrading from working 4.x.x to 5.x.x breaks the functionality

Recommendation for ARM64/Snapdragon users:

If you have AWS VPN Client 4.x.x working - DO NOT upgrade
If you need to install fresh, you'll need to find AWS VPN Client 4.x.x installer (not the latest 5.x.x)
Install OpenVPN ARM64 first, then AWS VPN Client 4.x.x

It appears AWS made changes in version 5.x.x that broke compatibility with third-party TAP drivers on ARM64. Until AWS provides native ARM64 support, staying on 4.x.x is the only option for Snapdragon devices.
Has anyone found a workaround for version 5.x.x?"
